src/utils/component-versioning.js

```diff
@@ -18,24 +18,33 @@ const extractDependencies = componentClass => {
 const oldTag = (tagName, closing = '') => `<${closing}${tagName}`;
 
 const newTag = (tagName, aVersion, closing) =>
-  oldTag(versionedTag(tagName, aVersion), closing);
+  oldTag(`${versionedTag(tagName, aVersion)} `, closing);
```

Comment:
I think the changes in component-versioning.js are too brittle and based on strong assumptions about HTML formatting.

This warrants its own principled treatment, see my attempt in https://github.com/axa-ch/patterns-library/pull/2051.

If my attempt is approved, you should undo your component-versioning.js work here, and simply merge after my PR.